app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt

```diff
@@ -227,18 +231,34 @@ fun HikeDetailScreen(
                   .padding(bottom = MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT + 8.dp))
 
       // Hike Details bottom sheet
-      HikeDetails(detailedRoute, savedHikesViewModel, elevationData, profile.hikingLevel)
+      HikeDetails(
+          detailedRoute = detailedRoute,
+          savedHikesViewModel = savedHikesViewModel,
+          elevationData = elevationData,
+          userHikingLevel = profile.hikingLevel,
+          onRunThisHike = { navigationActions.navigateTo(Screen.RUN_HIKE) },
```

Comment:
Question: I'm not sure to understand why you pass this as a parameter instead of just setting the `onClick` of the button ?

Reply:
Yes okay. Seems fair. I forgot that you would have to pass the `navigationActions`. Good to me then!